docs/apis-tools/go-client/go-get-started.md

```diff
@@ -22,22 +25,22 @@ First, we need a new Go project. To do this, complete the following steps:
 1. Create a new project using your IDE, or create a new Go module with the following command:
 
 ```
-mkdir -p $GOPATH/src/github.com/zb-user/zb-example
-cd $GOPATH/src/github.com/zb-user/zb-example
-go mod init
+mkdir github.com/zb-user/zb-example
+cd github.com/zb-user/zb-example
+go mod init zb-user/zb-example
 ```
 
 2. To use the Zeebe Go client library, add the following dependency to your `go.mod`:
 
 ```
 module github.com/zb-user/zb-example
 
-go 1.17
+go 1.20
```

Comment:
ATM, we are [using go 1.19](https://github.com/camunda/zeebe/blob/1c667b26030d76b8679948a8f5e20e65ffefaa78/clients/go/go.mod#L3C2-L3C2). Since I'm not a go expert; not sure if this makes a big difference

Reply:
```suggestion
go 1.19
```